sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/BlobChangeFeedClient.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.ChangeFeed.Models;
+
+namespace Azure.Storage.Blobs.ChangeFeed
+{
+    /// <summary>
+    /// BlobChangeFeedClient.
+    /// </summary>
+    public class BlobChangeFeedClient
```

Comment:
Types with client suffix need to have public ctors. i.e. "Client" means a type that users can start with.

Reply:
We could add public constructors.  Another option would be to make `GetChanges()` an extension method to `BlobServiceClient`, but we won't be able to do the same for Java and Python.

Reply:
The two options are: a) a public ctor, b) rename the type such that it does not have a "Client" suffix.

Reply:
I opted to add the public constructors, I figure there's no disadvantage to allowing customers to new up a `BlobChangeFeedClient` directly.